### ISSUE TITLE
MultiPolygonField, PolygonField has no attribute "srid"

### ISSUE THREAD

**Issue by StefanBrand** [typeddjango/django-stubs#1325]
# Bug report

## What's wrong
```py
error: Item "MultiPolygonField" of "Union[MultiPolygonField, PolygonField]" has no attribute "srid"  [union-attr]
error: Item "PolygonField" of "Union[MultiPolygonField, PolygonField]" has no attribute "srid"  [union-attr]
```

## How is that should be

At runtime, `PolygonField` has a `srid` attribute that is of type `int`.

```py
>>> type(self.geometry_field.srid)
<class 'int'>
```
Also see the Django source, where `BaseSpatialField` has the `srid` attribute:

https://github.com/django/django/blob/main/django/contrib/gis/db/models/fields.py#L72-L104

## System information

- OS: Docker image `python:3.9.16-bullseye`
- `python` version: 3.9.16
- `django` version: 4.1.4
- `mypy` version: 0-991
- `django-stubs` version: 1.13.1

**Comment by intgr** [comment]
Maybe @ciscorn is interested in tackling this?

**Comment by ciscorn** [comment]
@intgr @StefanBrand 

[My PR](https://github.com/typeddjango/django-stubs/pull/1299) may have possibly solved this problem, but it hasn't been released yet.

https://github.com/typeddjango/django-stubs/compare/master...ciscorn:django-stubs:976fb50#diff-c8252b7fb7ea667d89428fd3e9de78dca31e47d856b4fa4f69f2443286384a30R38

**Comment by intgr** [comment]
Ah, great, sorry :smile:

I will wrap up a release shortly.

**Comment by intgr** [comment]
Released version 1.13.2: https://github.com/typeddjango/django-stubs/releases/tag/1.13.2

Closing. Let me know if there are still any issues.
